liveblog.php

```diff
@@ -1221,6 +1381,16 @@ private static function is_wp_too_old() {
 		}
 		return version_compare( $wp_version, self::min_wp_version, '<' );
 	}
+
+	/**
+	 * Checks to see if the current WordPress version has REST API support
+	 *
+	 * @return bool true if supported, false otherwise
+	 */
+	private static function can_use_rest_api() {
+		global $wp_version;
+		return version_compare( $wp_version, self::min_wp_rest_api_version, '>=' );
```

Comment:
I did originally consider going that route but then opted for the version check as it seemed to make the most sense. I'm curious what your thoughts are on the benefits of checking for the existence of a specific class.